### PR TITLE
Expand docs

### PR DESCRIPTION
**Pull request by e-koch**
Update and expand the docs. And fix a few minor issues in the docs build.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,14 +1,37 @@
 Radio Beam
 ==========
 
-A tool for manipulating and utilizing two dimensional gaussian beams within the
-`astropy <http://www.astropy.org>`__ framework.
+radio-beam provides a tools for manipulating and utilizing two dimensional Gaussian beams
```
Comment (keflavich) [inline docs/index.rst:4]:
```suggestion
radio-beam provides a tools for manipulating and utilizing two-dimensional Gaussian beams
```

File: docs/index.rst [hunk 2]
```diff
@@ -1,14 +1,37 @@
...
+within the `astropy <http://www.astropy.org>`__ framework. It is primarily built for handling
+radio astronomy data and is integrated into the `spectral-cube <https://spectral-cube.readthedocs.io>_`
+package, amongst others.
 
+radio-beam also handles operations on sets of beams, for example from a spectral cube with
+varying resolution in spectral channels. Of note is the algorithms for identifying the
```
Comment (keflavich) [inline docs/index.rst:10]:
```suggestion
varying resolution in spectral channels. Of note are the algorithms for identifying the
```

File: docs/index.rst [hunk 3]
```diff
@@ -17,68 +40,138 @@ Read a beam from a fits header::
...
     >>> (1*u.Jy).to(u.K, u.brightness_temperature(25*u.GHz, my_beam)) # doctest: +FLOAT_CMP
     <Quantity 7821.572919292681 K>
 
-Convolve with another beam::
+Or alternatively with::
+
+    >>> (1*u.Jy).to(u.K, my_beam.jtok_equiv(25*u.GHz)) # doctest: +FLOAT_CMP
+    <Quantity 7821.572919292681 K>
+
+To get the value off 1 Jy in K for a given beam::
```
Comment (keflavich) [inline docs/index.rst:84]:
```suggestion
To get the value of 1 Jy in K for a given beam::
```

File: docs/index.rst [hunk 4]
```diff
@@ -17,68 +40,138 @@ Read a beam from a fits header::
...
 
     >>> from radio_beam import Beams
     >>> from astropy.io import fits
     >>> bin_hdu = fits.open('file.fits')[1]  # doctest: +SKIP
     >>> beams = Beams.from_fits_bintable(bin_hdu)  # doctest: +SKIP
 
+In the above example, the second FITS extension contains the beam tables, while the
+first would have the spectral cube data.
+
+To read a table of beams from a CASA image (must be run inside a CASA environment!)::
```
Comment (keflavich) [inline docs/index.rst:127]: can we do this w/casa-formats-io now?

**Review by keflavich — APPROVED** [review]

**Comment by keflavich** [comment]
merging as soon as CI passes

File: docs/index.rst (hunk 4 above)
Comment (e-koch) [reply docs/index.rst:127]: Probably. I raised an issue: https://github.com/radio-astro-tools/casa-formats-io/issues/42.

**Comment by e-koch** [comment]
@keflavich ready to merge.

**Pull request merged**